src/els_config.erl

```diff
@@ -23,6 +23,13 @@
 %%==============================================================================
 -define(DEFAULT_CONFIG_PATH, "erlang_ls.config").
 -define(SERVER, ?MODULE).
+-define( DEEFAULT_EXCLUDED_OTP_APPS
```

Comment:
Typo: DEEFAULT

Reply:
Also, please keep them in lexicographical order.

Reply:
🤦‍♂ Fixed